packages/plugin-packages/mashroom-websocket/package.json

```diff
@@ -56,6 +57,7 @@
                 ],
                 "defaultConfig": {
                     "path": "/websocket",
+                    "tmpFileStorePath": "/tmp",
```

Comment:
I'm not completely happy with this name because it sounds very generic. A better name would be. "reconnectMessageBufferFolder" or something like this. Also I would set it to "null" by default because like this if someone updates to the latest version and /tmp folder ist not writable the plugin doesn't work anymore... And if the property is set to null, buffering and reconnect should be disabled. In that warning could be put into the log.